demo/readme.md

```diff
@@ -90,21 +90,20 @@ The demos begin with a fully [configured Kubernetes cluster](https://kubernetes.
 ### Demo steps
 
 1. Validate the status of the [Kubernetes cluster](https://kubernetes.io/docs/setup/independent/create-cluster-kubeadm/).
-2. Clone the [Intel Device Plugins for Kubernetes source](https://github.com/intel/intel-device-plugins-for-kubernetes).
```

Comment:
is this a miss?

Reply:
OK, I see this is indeed desired but we may want to fix 1...3, 4, 5

Reply:
@mythi thank you for the review. updated the PR.